### DISCUSSION TITLE
[Bug]: Glitch in 'Create list' button

### DISCUSSION THREAD

**Discussion by gupta-ji6** [community/community#8646] [Other Feature Feedback, Questions, & Ideas] [answered]
The "Create List" button is not responsive and glitches as shown in the video below.
It worked as expected when I clicked on the far right of the button.

Repo I faced the glitch on - https://github.com/magicmark/apollo-error-handling-visualizer

https://user-images.githubusercontent.com/21218732/145538866-d27029b2-ac58-4994-b812-ba0b8cc7c2da.mov

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up — we've rolled out a fix that was causing this button to not trigger the Create List dialog, would you be up for trying again to see if things are working correctly now?

**Reply by gupta-ji6** [reply]
works as expected now, thanks! 👌🏻

**Reply by yehiamitro111** [reply]
Thankyou

**Comment by ghost** [comment]


**Reply by rmathew1992** [reply]
Hello! I've started to investigate what might be going on here, and was hoping for some additional info as I wasn't able to reproduce this on firefox. Could you let me know what device you're using? Thanks!

**Reply by ghost** [reply]
It looks like something is wrong in my prefs.js settings, sorry.

**Reply by ghost** [reply]
`lockPref("dom.enable_performance_observer", false);`

**Reply by ghost** [reply]
```
lockPref("dom.enable_performance", false);
lockPref("dom.enable_performance_observer", false);
lockPref("dom.enable_performance_navigation_timing", false);
```

**Reply by zhukovgreen** [reply]
Still happen to me
